src/app/api/api_v1/endpoints/detections.py

```diff
@@ -128,6 +132,40 @@ async def fetch_detections(
     return filtered_detections
 
 
+@router.get("/unacknowledged/from", status_code=status.HTTP_200_OK, summary="Fetch all the unacknowledged detections")
+async def fetch_unacknowledged_detections(
+    from_date: Optional[datetime] = None,
+    detections: DetectionCRUD = Depends(get_detection_crud),
+    cameras: CameraCRUD = Depends(get_camera_crud),
+    token_payload: TokenPayload = Security(get_jwt, scopes=[UserRole.ADMIN, UserRole.AGENT, UserRole.USER]),
+) -> List[Detection]:
+    telemetry_client.capture(token_payload.sub, event="unacknowledged-fetch")
+
+    try:
+        all_unck_detections = [elt for elt in await detections.fetch_all() if elt.is_wildfire is None]
+        if from_date is not None:
+            all_unck_detections = [detection for detection in all_unck_detections if detection.created_at >= from_date]
+    except Exception as e:  # noqa
```

Comment:
I usually avoid this: if this throws an error, let's leave it to sentry. It will be much easier to debug

Otherwise let's have very specific exceptions

Reply:
I don't have sentry in development. In this case I could create a loggger which send to sentry and terminal ?

Reply:
But the point is catching that in prod, isn't it? I don't get what you mean by "in development"
To me, we only want to catch things that we know specifically that can happen. but a wide `Exception` will catch anything, even things we don't know and would like to fix.
Considering the worker restarts in a clean way when it crashes, I prefer this option because we get the appropriate signal (sentry notif with trace & everything). So we can log some information but my problem is more with the `except Exception`

Reply:
I raise the exception after (L140)

When I develop I have always lots of trouble to debug because I don't see error in the log